File: Projekti/C#-Native-Benchmark/Repository/ProfileRepository.cs
using MySql.Data.MySqlClient;
using Native.Repository.Interface;
using Native.Settings;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Native.Repository
{
    public class ProfileRepository : IProfileRepository
    {
        private readonly string _connectionString;

        public ProfileRepository(IDatabaseConnection connection)
        {
            _connectionString = connection.ConnectionString;
        }

        public void DeleteProfilesWithPhoneNumber(string phoneNumber)
        {
            using MySqlConnection connection = new(_connectionString);
            connection.Open();
            MySqlCommand command = connection.CreateCommand();

            command.CommandText = "DELETE FROM Profiles WHERE Profiles.phone_number LIKE @param";
            command.Parameters.AddWithValue("@param", phoneNumber + "%");
            command.ExecuteNonQuery();
        }
    }
}

File: Projekti/Python-Native-Benchmark/repository/profile_repository.py
class ProfileRepository:
    def __init__(self, conn):
        self.conn = conn

    def delete_profiles_by_phone_number(self, number):

        query = "DELETE FROM Profiles WHERE Profiles.phone_number LIKE %s"

        with self.conn.cursor() as cursor: 
            cursor.execute(query, (number + "%",))

        self.conn.commit()